bindings/nodejs/types/bridge/account.ts

```diff
@@ -135,9 +136,9 @@ export type __PrepareMintNftsMethod__ = {
 export type __PrepareTransactionMethod__ = {
     name: 'PrepareTransaction';
     data: {
-        outputs: Output[];
+        outputs: OutputTypes[];
```

Comment:
Is this really correct? The name at least sounds not really correct

Reply:
I think so.  This is the naming convention for union types in `@iota/types`.

https://github.com/iotaledger/iota.js/blob/feat/stardust/packages/iota/src/models/outputs/outputTypes.ts

Reply:
I agree that it could be called just `Output` though.